rules/windows/process_creation/win_susp_process_creations.yml

```diff
@@ -24,6 +25,17 @@ logsource:
 detection:
     selection:
         CommandLine:
+            - vssadmin.exe delete shadows*
+            - vssadmin delete shadows*
+            - vssadmin create shadow /for=C:*
+            - copy \\?\GLOBALROOT\Device\\*\windows\ntds\ntds.dit*
+            - copy \\?\GLOBALROOT\Device\\*\config\SAM*
+            - copy \\?\GLOBALROOT\Device\\*\config\SYSTEM*
+            - type \\?\GLOBALROOT\Device\\*\config\SAM*
+            - type \\?\GLOBALROOT\Device\\*\config\SYSTEM*
+            - type \\?\GLOBALROOT\Device\\*\windows\ntds\ntds.dit*
```

Comment:
duplicate with [rules/windows/process_creation/win_copying_sensitive_files_with_credential_data.yml](https://github.com/Neo23x0/sigma/blob/oscd/rules/windows/process_creation/win_copying_sensitive_files_with_credential_data.yml), that is more resource-optimized and contextualized.